docs/admin/index.md

```diff
@@ -72,3 +74,21 @@ explicitly-managed secret.
 
 [web-platform-tests]: https://github.com/e3c/web-platform-tests
 [wpt.fyi]: https://github.com/web-platform-tests/wpt.fyi
+
+## Emergency playbook
+
+### Lock down write access to the repo
+
+**Recommended but not yet verified approach:** Create a [new branch protection
+rule](https://github.com/web-platform-tests/wpt/settings/branch_protection_rules/new)
+that applies to `*` (i.e. all branches), and check "Restrict who can push to
+matching branches". This should prevent everyone except those with the
+"Maintain" role (currently only the GitHub admins listed above) from pushing
+to *any* branch. To lift the limit, delete this branch protection rule.
+
+**Alternative apporach proven to work in [#21424]
+(https://github.com/web-platform-tests/wpt/issues/21424):** Go to [manage
+access](https://github.com/web-platform-tests/wpt/settings/access), and
+change the permission of "reviewers" to "Read". To lift the limit, change it
+back to "Write". This has the known downside of *resubscring all reviewers
+to repo notifications*.
```

Comment:
**Alternative apporach proven to work in
[#21424](https://github.com/web-platform-tests/wpt/issues/21424):** Go to
[manage access](https://github.com/web-platform-tests/wpt/settings/access),
and change the permission of "reviewers" to "Read". To lift the limit, change
it back to "Write". This has the known downside of *resubscring all reviewers
to repo notifications*.
```